cluster/environments/azure-multiple-clusters-waf-tm-apimgmt/aks-centralus.tf

```diff
@@ -31,7 +31,8 @@ module "central_vnet" {
 
 # Creates aks cluster
 module "central_aks" {
-  source = "github.com/Microsoft/bedrock/cluster/azure/aks"
+  //source = "github.com/Microsoft/bedrock/cluster/azure/aks"
+  source = "../../azure/aks"
```

Comment:
@yradsmikham I know there is probably a reason why we changed back to relative references. Can you help me understand why?

Reply:
I changed it the relative path just so that it will pass the integration tests on Azure DevOps with the new changes. Once it passes (and it appears to have passed now), I will change it back to the original source.